### PR TITLE
Provisioning

### PR DESCRIPTION
**Pull request by dlabaj**
## Motivation
Updated with PF4 Styling for provisioning

## Verification Steps

1. Loaded provisioning screen

## Checklist:

- [ X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

screen shot 2019-02-19 at 11 37 02 pm

screen shot 2019-02-19 at 11 38 49 pm

### CODE REVIEW

File: src/styles/application/pages/_provisioning.scss [hunk 1]
```diff
@@ -144,6 +140,40 @@
...
+.integr8ly-provisioning-check {
+  color: $pf-color-green-300;
+}
+
+.integr8ly-provisioning-spinner {
+  color: $pf-color-blue-400;
+}
+
+.integr8ly-provisioning-text {
+  margin-left: 8px;
```
Comment (mindreeper2420) [inline src/styles/application/pages/_provisioning.scss]: Remove and replace with the PatternFly 4 spacer utility.

File: src/components/provisioning/provisioning.js [hunk 2]
```diff
@@ -69,17 +77,30 @@ function buildProvisioningScreen(WrappedComponent) {
...
       return null;
     }
 
     static renderServiceLoadingIcon(svc) {
       if (isServiceProvisioned(svc)) {
-        return <i className="pficon pficon-on-running integr8ly-status-ready" />;
+        return (
+          <div>
+            <CheckCircleIcon className="integr8ly-provisioning-check" />
+            <span className="integr8ly-provisioning-text"> Ready to use</span>
```
Comment (mindreeper2420) [inline src/components/provisioning/provisioning.js]:
```suggestion
            <span className="integr8ly-provisioning-text pf-u-ml-sm"> Ready to use</span>
```

File: src/styles/application/pages/_provisioning.scss (hunk 1 above)
Comment (dlabaj) [reply src/styles/application/pages/_provisioning.scss]: Done

File: src/components/provisioning/provisioning.js (hunk 2 above)
Comment (dlabaj) [reply src/components/provisioning/provisioning.js]: Done

**Pull request merged**